linkerd/docs/config.md

```diff
@@ -270,3 +271,15 @@ Key | Default Value | Description
 --- | ------------- | -----------
 orgId | empty by default | Optional string of your choosing that identifies your organization
 enabled | true | If set to true, data is sent to Buoyant once per hour
+
+### Socket Options
+
+Linkerd supports configuring socket level options for any given interface.
+i.e. the admin and router interfaces. These configurations are only available 
+on Linux 3.9 distributions and newer.
+
+Key | Default Value | Description
+--- | ------------- | -----------
+noDelay | true | If set to true, disables the use of `TCP_NODELAY` on a socket interface
```

Comment:
I think this is backwards.  If set to true it enables TCP_NODELAY

Reply:
According the the docs in Finagle's Transport.scala it looks like if it is set to true it is disabled and enabled if it is false.

```
/**
   * $param the options (i.e., socket options) of a `Transport`.
   *
   * @param noDelay enables or disables `TCP_NODELAY` (Nagle's algorithm)
   *                option on a transport socket (`noDelay = true` means
   *                disabled). Default is `true` (disabled).
   *
   * @param reuseAddr enables or disables `SO_REUSEADDR` option on a
   *                  transport socket. Default is `true`.
   *
   * @param reusePort enables or disables `SO_REUSEPORT` option on a
   *                  transport socket (Linux 3.9+ only). This option is only
   *                  available when using finagle-netty4 and native epoll support
   *                  is enabled. Default is `false`.
   */
```